src/components/AddTodoForm.js

```diff
@@ -1,14 +1,35 @@
-import React from "react";
+import React, { useState } from "react";
+import TodoList from "./TodoList";
 
 const AddTodoForm = () => {
+  const [TodoItem, setTodoItem] = useState("");
+  // By Bellow code we take the value of input feild and set it back to TodoItem using useState.
+  const handelChange = (event) => {
+    setTodoItem(event.target.value);
+  };
+
+  const handleAddTodo = (event) => {
+    event.preventDefault();
+    console.log(event.target.title.value);
```

Comment:
`event.target.title.value` is correct way to get the new todo. Don't forget that you need to do something else with the value beside console.logging it before you reset it to `""`